docker/linea-mainnet/docker-compose.yml

```diff
@@ -69,3 +69,41 @@ services:
       - ./besu-genesis.json:/var/lib/besu/genesis.json:ro
       - ../config/linea-besu-sequencer/log4j.xml:/var/lib/besu/log4j.xml
       - ./static-nodes.json:/var/lib/besu/static-nodes.json
+
+  #Erigon node
+  erigon-init:
+    image: erigontech/erigon:2.60.8
+    command:
+      - init
+      - /erigon-genesis.json
+      - --datadir=/data
+    volumes:
+      - ./genesis.json:/erigon-genesis.json:ro
+      - /data/linea-datadir:/home/erigon/.local/share/erigon/
```

Comment:
I would make it a docker named volume here; this would avoid polluting the local filesystem, WDYT?

Reply:
100%!  I’ve updated the volume to use a Docker named volume for each clients.